client/package.json

```diff
@@ -49,7 +49,8 @@
     "react-html-parser": "^2.0.2",
     "react-lazyload": "^2.6.2",
     "react-on-rails": "11.3.0",
-    "react-render-html": "^0.6.0"
```

Comment:
We can undo these changes since we're not using these dependencies. Same goes for the `yarn.lock` changes.

Reply:
Dear Julia, I will be very glad to undo it but I guess they came automatically from when I pulled updates from Master 😅. I wouldn't know how to und them 🤨. Any help?

Reply:
You'll want to do a soft reset on the commit that has those changes and remove them from it. If that commit isn't the latest one. You can just cherry-pick the commits that are on top of that one.

Alternatively, you could just squash everything in one commit and do the soft reset on that.

Reply:
Could we also use `git revert`?
From what I understand `reset` would lose the later commits and could lead to some conflicts with the memory/potential loss of code.